api/v1beta1/foundationdbcluster_types.go

```diff
@@ -296,6 +296,14 @@ type FoundationDBClusterSpec struct {
 	// investigating in issues or if the environment is unstable.
 	// +kubebuilder:default:=false
 	Skip bool `json:"skip,omitempty"`
+
+	// CoordinatorSelection defines which process classes are eligible for coordinator selection.
+	// If empty all stateful processes classes are equally eligible.
+	// A higher priority means that a process class is preferred over another process class.
+	// If the FoundationDB cluster is sans across multiple Kubernetes clusters or DCs the
+	// CoordinatorSelection must match in all FoundationDB cluster resources otherwise
+	// the coordinator selection process could conflict.
+	CoordinatorSelection []CoordinatorSelectionSetting `json:"coordinatorSelection,omitempty"`
```

Comment:
The word `sans` doesn't sound right here, but I'm not sure what word you were going for.

Reply:
🤦 that should be `spans`